### ISSUE TITLE
Fatal Error: 'fmdb/FMDB.h' file not found when building for iOS

### ISSUE THREAD

**Issue by AlabasterAxe** [tekartik/sqflite#461]
I get the following error when I attempt to build for iOS:

/Volumes/SSD/development/flutter/.pub-cache/hosted/pub.dartlang.org/sqflite-1.3.1/ios/Classes/SqflitePlugin.m:3:9: fatal error: 'fmdb/FMDB.h' file not found
    #import <fmdb/FMDB.h>

I definitely see FMDB in my list of installed pods:
![Screen Shot 2020-06-17 at 4 58 23 PM](https://user-images.githubusercontent.com/1403638/84949960-14f28600-b0bc-11ea-9eaf-2c569587d79e.png)

My project, Flutter installation and XCode all live on an external drive that are permanently attached to my Mac

**Things I've tried:**
 - Flutter clean
 - Deleting my cached pub packages
 - Uninstalling and reinstalling flutter
 - completely deleting my project and pulling again and building from scratch
 - running flutter create . on my project

**flutter doctor -v output:**
[✓] Flutter (Channel stable, v1.17.3, on Mac OS X 10.15.5 19F101, locale en-US)
    • Flutter version 1.17.3 at /Volumes/SSD/development/flutter
    • Framework revision b041144f83 (13 days ago), 2020-06-04 09:26:11 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4

[!] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/matthewkeller/Library/Android/sdk
    • Platform android-29, build-tools 29.0.3
    • Java binary at: /usr/bin/java
    • Java version Java(TM) SE Runtime Environment (build 14.0.1+7)
    ✗ Android license status unknown.
      Try re-installing or updating your Android SDK Manager.
      See https://developer.android.com/studio/#downloads or visit visit
      https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Volumes/SSD/Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.8.4

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).

[✓] VS Code (version 1.46.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.11.0

[✓] Connected device (1 available)
    • iPad Pro (12.9-inch) (4th generation) • 830FDFFD-86C8-42A4-A0C4-24C104FB71FA • ios •
      com.apple.CoreSimulator.SimRuntime.iOS-13-5 (simulator)

**Comment by alextekartik** [comment]
These pods issues are killing me...

Some suggestions:
* Have you tried to force a `pod install` / `pod update`
* Try to build the example app here: https://github.com/tekartik/sqflite/tree/master/sqflite/example and see if it works
* If you generated your iOS project a long time ago, you could try to regenerate you ios project (delete existing ios project and perform a `flutter create` again)
* Create a new project, add sqflite dependency and check that it compiles

**Comment by AlabasterAxe** [comment]
Awesome, thanks for the quick reply. I'll give these things a shot and let you know.

**Comment by AlabasterAxe** [comment]
Got a chance to try a few of these things. Pretty confusing results:

- I `pod install`ed and `pod update`d, no observable change
- I cloned your repo and it built and ran fine
- I deleted and regenerated my iOS directory, it built but then insta-crashed when I launched it

Not really sure where to go from here but it doesn't look specific to sqflite. If you have any other suggestions, I'd definitely be interested to hear them but feel free to close this.

**Comment by alextekartik** [comment]
> I deleted and regenerated my iOS directory, it built but then insta-crashed when I launched it

Good to figure out why it is crashing here as it is maybe a small issue. I see that you are using firebase so you likely need to re-add google services json file (I know it crashes when not present)

**Comment by AlabasterAxe** [comment]
Thanks for your info and quick replies. I believe I tried a couple of variants of deleting the iOS directory some of which kept the old GoogleServices.plist and it still crashed. BTW, when it crashed it didn't print any logs from my flutter code so it seemed like it crashed before it got a chance to execute any of my Dart code, could be wrong though.

When I was getting this error I had my project on an external drive. I've since moved my project back to the internal SSD of my machine and it's building again. Not a particularli enlightening solution but mostly just wanted to get this back and building.
